docs/podman-build.1.md

```diff
@@ -207,11 +207,19 @@ that the IPC namespace in which `buildah` itself is being run should be reused,
 or it can be the path to an IPC namespace which is already in use by
 another process.
 
-**--isolation** [Not Supported]
-
-Podman is not currently supported on Windows, and does not have a daemon.
-If you want to override the container isolation you can choose a different
-OCI Runtime, using the --runtime flag.
+**--isolation** *type*
```

Comment:
Has the version of Buildah with this option in it been vendored in to Podman?

Reply:
Yes